atlasdb-cassandra/src/main/java/com/palantir/atlasdb/keyvalue/cassandra/CassandraClientPoolImpl.java

```diff
@@ -229,6 +210,20 @@ public void shutdown() {
         metrics.deregisterMetrics();
     }
 
+    /**
+     * This is the maximum number of times we'll accept connection failures to one host before blacklisting it. Note
+     * that subsequent hosts we try in the same call will actually be blacklisted after one connection failure
+     */
+    @VisibleForTesting
+    int getMaxRetriesPerHost() {
```

Comment:
I assume these will eventually just live on `CassandraRequestExceptionHandler` (and possibly be set via config?)

Reply:
I think that's the long term intention. Eventually these will be suppliers injected from the config.